packages/gestalt/src/Dropdown.js

```diff
@@ -122,10 +125,18 @@ type Props = {|
    *  Define a controlled size to dropdown's Popover.
    */
   maxHeight?: '30vh',
+  /**
+   * Mobile-only prop. Callback fired when Dropdown's in & out animations end. See the [mobile variant](https://gestalt.pinterest.systems/web/dropdown#mobile) to learn more.
+   */
+  mobileOnAnimationEnd?: ({| animationState: 'in' | 'out' |}) => void,
   /**
    * Callback fired when the menu is closed.
    */
   onDismiss: () => void,
+  /**
+   * Dropdown can adapt to mobile devices to [SheetMobile](https://gestalt.pinterest.systems/web/sheetmobile). Mobile adaptation is not enabled by default. Set to 'sheetMobile' to enable SheetMobile in mobile devices. See the [mobile variant](https://gestalt.pinterest.systems/web/dropdown#mobile) to learn more.
+   */
+  mobileAdaptation?: 'default' | 'sheetMobile',
```

Comment:
Is the plan to eventually make `sheetMobile` the default? If so, we may want to make this something like `disableMobileUI` and codemod all current usages to include the prop, such that any new usages won't have it — therefore encouraging new Dropdown usages to embrace the adaptive UI.

Reply:
DONE

Keeping disabled, I'll run codemod next and invert default as a follow up. less blocking PR